apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx

```diff
@@ -24,36 +48,70 @@ export const Billing = () => {
                     </div>
                     <div className="space-y-6">
                         {planDetails.map(({ icon: Icon, ...plan }) => {
+                            const count =
```

Comment:
@afolasope Can you simplify this conditional statement? It's very difficult to. read